scripts/cibuild

```diff
@@ -70,6 +72,7 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         echo "Building app, airflow..."
         GIT_COMMIT="${GIT_COMMIT}" docker-compose \
             -f "${DIR}/../docker-compose.yml" \
+            -f "${DIR}/../docker-compose.airflow.yml" \
```

Comment:
This task doesn't seem to reference any Airflow components, so I don't think we need to include `docker-compose.airflow.yml` here.